src/almanac/introduction.md

```diff
@@ -1,3 +1,72 @@
 # Welcome
 
 Welcome to the Software Gardening Almanac, an open-source handbook of applied guidance and tools for sustainable software development and maintenance.
+
+## Inspiration
+
+```{figure} assets/software-lifecycle.png
+---
+alt: A software file shown in sequences with symbols indicating creation, growth,
+  and decay.
+---
+_Software is created, grows, and decays over time._
+```
+
+Software undergoes time-based cycles phases that are not well understood or cultivated.
+Why does software grow quickly only to decay just as fast?
+How do software bugs seem to appear in the most unlikely scenarios?
+
+These cycles follow patterns from life: software is created, grows, decays, and so on (sometimes in surprising or seemingly unpredictable ways).
+Software is also connected within a complex system of relationships (similar to ecology).
+The Software Gardening Almanac posits we can understand these software lifecycle patterns and complex relationships in order to build tools which sustain or maintain their development long-term.
+
+> _"The ‘planetary garden’ is a means of considering ecology as the integration of humanity – the gardeners – into its smallest spaces._
+> _Its guiding philosophy is based on the principle of the ‘garden in motion’: do the most __for__, the minimum __against__."_
+> \- Gilles Clément
+
+The content here is inspired by ecological systems (for example, as in [_planetary gardening_](https://www.architectural-review.com/essays/in-practice/in-practice-gilles-clement-on-the-planetary-garden)).
+We also are galvanized by the [_scientific method_](https://en.wikipedia.org/wiki/Scientific_method), and [_almanacs_](https://en.wikipedia.org/wiki/Almanac) (or earlier [_menologia rustica_](https://en.wikipedia.org/wiki/Menologia_rustica)) in documenting, expecting, and optimizing how we plan for time-based changes in agriculture (among other practices and traditions).
+
+```{figure} assets/almanac-influencing-software.png
+---
+height: 270px
+alt: A book shown as influencing software within the cycle of time.
+---
+_Almanacs help us understand and influence the impacts of time on the things we grow._
+```
+
+Almanacs within the context of computers can be interpreted as meta-software which use objective, shared experience to help encourage growth and avoid unwanted decay.
```

Comment:
I'm a bit confused by this sentence - maybe delete and get right to the point?

Reply:
For example, what is meta-software? what do you mean by unwanted decay? Is decay ever wanted?

Reply:
Thanks, good points, I'll see about revising this a bit more. Generally I feel that decay is an important aspect to witness as software emerges (sometimes we incrementally improve through the decay, and it feels necessary).